packages/datadog-plugin-mysql/src/index.js

```diff
@@ -22,6 +22,12 @@ class MySQLPlugin extends DatabasePlugin {
         'out.port': dbConfig.port
       }
     })
+    if (this.config.dbmPropagationMode !== 'disabled') {
+      if (sqlStatement[0].sql !== undefined) {
+        const key = 'sql'
+        sqlStatement[0][key] = this.injectDbmQuery(sqlStatement[0].sql)
```

Comment:
Just an FYI, if #2700 merges first, the `injectDbmQuery()` calls will need `service` passed in as a second arg. If this PR merges first I'll update #2700.